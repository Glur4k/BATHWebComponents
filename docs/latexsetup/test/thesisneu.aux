\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@input{includes/cover.aux}
\@input{includes/title.aux}
\@input{includes/abstract.aux}
\@input{includes/affidavit.aux}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Custom Elements}{5}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{custom-elements}{{1}{5}{Custom Elements}{chapter.1}{}}
\@writefile{toc}{\contentsline {section}{Einf\IeC {\"u}hrung}{5}{section*.2}}
\newlabel{einfuxfchrung}{{1}{5}{Einführung}{section*.2}{}}
\@writefile{toc}{\contentsline {section}{Neue Elemente registrieren}{6}{section*.3}}
\newlabel{neue-elemente-registrieren}{{1}{6}{Neue Elemente registrieren}{section*.3}{}}
\@writefile{toc}{\contentsline {section}{Vorteile von Custom Elements}{7}{section*.4}}
\newlabel{vorteile-von-custom-elements}{{1}{7}{Vorteile von Custom Elements}{section*.4}{}}
\@writefile{toc}{\contentsline {subsection}{Nachteil}{7}{section*.5}}
\newlabel{nachteil}{{1}{7}{Nachteil}{section*.5}{}}
\@writefile{toc}{\contentsline {section}{Vorhandene Elemente erweitern (Type extensions)}{7}{section*.6}}
\newlabel{vorhandene-elemente-erweitern-type-extensions}{{1}{7}{Vorhandene Elemente erweitern (Type extensions)}{section*.6}{}}
\@writefile{toc}{\contentsline {subsection}{Verwendung bei Github}{8}{section*.7}}
\newlabel{verwendung-bei-github}{{1}{8}{Verwendung bei Github}{section*.7}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.1}{\ignorespaces Bild: Github Einsatz eines Custom Element}}{8}{figure.1.1}}
\@writefile{toc}{\contentsline {section}{Eigenschaften und Methoden definieren}{9}{section*.8}}
\newlabel{eigenschaften-und-methoden-definieren}{{1}{9}{Eigenschaften und Methoden definieren}{section*.8}{}}
\@writefile{toc}{\contentsline {section}{Custom Element Life Cycle Callbacks - TODO?}{9}{section*.9}}
\newlabel{custom-element-life-cycle-callbacks---todo}{{1}{9}{Custom Element Life Cycle Callbacks - TODO?}{section*.9}{}}
\@writefile{toc}{\contentsline {subsubsection}{createdCallback}{9}{section*.10}}
\newlabel{createdcallback}{{1}{9}{createdCallback}{section*.10}{}}
\@writefile{toc}{\contentsline {subsubsection}{attachedCallback}{9}{section*.11}}
\newlabel{attachedcallback}{{1}{9}{attachedCallback}{section*.11}{}}
\@writefile{toc}{\contentsline {subsubsection}{detachedCallback}{10}{section*.12}}
\newlabel{detachedcallback}{{1}{10}{detachedCallback}{section*.12}{}}
\@writefile{toc}{\contentsline {subsubsection}{attributeChangedCallback}{10}{section*.13}}
\newlabel{attributechangedcallback}{{1}{10}{attributeChangedCallback}{section*.13}{}}
\@writefile{toc}{\contentsline {subsection}{Beispiel mit \texttt  {button-extended}}{10}{section*.14}}
\newlabel{beispiel-mit-button-extended}{{1}{10}{\texorpdfstring {Beispiel mit \texttt {button-extended}}{Beispiel mit button-extended}}{section*.14}{}}
\@writefile{toc}{\contentsline {section}{Styling von Custom Elements}{10}{section*.15}}
\newlabel{styling-von-custom-elements}{{1}{10}{Styling von Custom Elements}{section*.15}{}}
\@writefile{toc}{\contentsline {section}{Browserunterst\IeC {\"u}tzung}{11}{section*.16}}
\newlabel{browserunterstuxfctzung}{{1}{11}{Browserunterstützung}{section*.16}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.2}{\ignorespaces Bild: Browserunterst\IeC {\"u}tzung von Custom Elements}}{11}{figure.1.2}}
\@writefile{toc}{\contentsline {section}{Quellen}{11}{section*.17}}
\newlabel{quellen}{{1}{11}{Quellen}{section*.17}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}Was anderes}{12}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{was-anderes}{{2}{12}{Was anderes}{chapter.2}{}}
