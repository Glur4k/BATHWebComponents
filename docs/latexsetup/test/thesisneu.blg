This is BibTeX, Version 0.99dThe top-level auxiliary file: thesisneu.aux
A level-1 auxiliary file: includes/cover.aux
A level-1 auxiliary file: includes/title.aux
A level-1 auxiliary file: includes/abstract.aux
A level-1 auxiliary file: includes/affidavit.aux
I found no \citation commands---while reading file thesisneu.aux
I found no \bibdata command---while reading file thesisneu.aux
I found no \bibstyle command---while reading file thesisneu.aux
(There were 3 error messages)
