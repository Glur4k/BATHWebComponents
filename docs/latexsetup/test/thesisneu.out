\BOOKMARK [0][-]{chapter*.1}{\376\377\000E\000h\000r\000e\000n\000w\000"\000o\000r\000t\000l\000i\000c\000h\000e\000\040\000E\000r\000k\000l\000"\000a\000r\000u\000n\000g}{}% 1
\BOOKMARK [0][-]{chapter.1}{\376\377\000C\000u\000s\000t\000o\000m\000\040\000E\000l\000e\000m\000e\000n\000t\000s}{}% 2
\BOOKMARK [1][-]{section*.2}{\376\377\000E\000i\000n\000f\000\374\000h\000r\000u\000n\000g}{chapter.1}% 3
\BOOKMARK [1][-]{section*.3}{\376\377\000N\000e\000u\000e\000\040\000E\000l\000e\000m\000e\000n\000t\000e\000\040\000r\000e\000g\000i\000s\000t\000r\000i\000e\000r\000e\000n}{chapter.1}% 4
\BOOKMARK [1][-]{section*.4}{\376\377\000V\000o\000r\000t\000e\000i\000l\000e\000\040\000v\000o\000n\000\040\000C\000u\000s\000t\000o\000m\000\040\000E\000l\000e\000m\000e\000n\000t\000s}{chapter.1}% 5
\BOOKMARK [2][-]{section*.5}{\376\377\000N\000a\000c\000h\000t\000e\000i\000l}{section*.4}% 6
\BOOKMARK [1][-]{section*.6}{\376\377\000V\000o\000r\000h\000a\000n\000d\000e\000n\000e\000\040\000E\000l\000e\000m\000e\000n\000t\000e\000\040\000e\000r\000w\000e\000i\000t\000e\000r\000n\000\040\000\050\000T\000y\000p\000e\000\040\000e\000x\000t\000e\000n\000s\000i\000o\000n\000s\000\051}{chapter.1}% 7
\BOOKMARK [2][-]{section*.7}{\376\377\000V\000e\000r\000w\000e\000n\000d\000u\000n\000g\000\040\000b\000e\000i\000\040\000G\000i\000t\000h\000u\000b}{section*.6}% 8
\BOOKMARK [1][-]{section*.8}{\376\377\000E\000i\000g\000e\000n\000s\000c\000h\000a\000f\000t\000e\000n\000\040\000u\000n\000d\000\040\000M\000e\000t\000h\000o\000d\000e\000n\000\040\000d\000e\000f\000i\000n\000i\000e\000r\000e\000n}{chapter.1}% 9
\BOOKMARK [1][-]{section*.9}{\376\377\000C\000u\000s\000t\000o\000m\000\040\000E\000l\000e\000m\000e\000n\000t\000\040\000L\000i\000f\000e\000\040\000C\000y\000c\000l\000e\000\040\000C\000a\000l\000l\000b\000a\000c\000k\000s\000\040\000-\000\040\000T\000O\000D\000O\000?}{chapter.1}% 10
\BOOKMARK [2][-]{section*.14}{\376\377\000B\000e\000i\000s\000p\000i\000e\000l\000\040\000m\000i\000t\000\040\000b\000u\000t\000t\000o\000n\000-\000e\000x\000t\000e\000n\000d\000e\000d}{section*.9}% 11
\BOOKMARK [1][-]{section*.15}{\376\377\000S\000t\000y\000l\000i\000n\000g\000\040\000v\000o\000n\000\040\000C\000u\000s\000t\000o\000m\000\040\000E\000l\000e\000m\000e\000n\000t\000s}{chapter.1}% 12
\BOOKMARK [1][-]{section*.16}{\376\377\000B\000r\000o\000w\000s\000e\000r\000u\000n\000t\000e\000r\000s\000t\000\374\000t\000z\000u\000n\000g}{chapter.1}% 13
\BOOKMARK [1][-]{section*.17}{\376\377\000Q\000u\000e\000l\000l\000e\000n}{chapter.1}% 14
\BOOKMARK [0][-]{chapter.2}{\376\377\000W\000a\000s\000\040\000a\000n\000d\000e\000r\000e\000s}{}% 15
